app/views/root/_publication.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if publication.published? %>
       <%= link_to "#{Plek.new.website_root}/#{publication.slug}", class: 'link-muted' %>
-    <% elsif publication.safe_to_preview? %>
+    <% elsif !publication.archived? %>
```

Comment:
There seems to be some subtlety for `safe_to_preview?` for GuideEditions (in content models) so you might have to keep it around. Possibly `publication.safe_to_preview? && !publication.archived?` _or_ update the `safe_to_preview?` method in content models to return false for archived content...?